.github/workflows/opendata-ci.yml

```
@@ -0,0 +1,42 @@
+name: Open Data CI
+on:
+  pull_request:
+    branches:
+      - dev
+    paths:
+      - 'src/open-data/**/*.csv'
+      - 'src/open-data/**/*.json'
```

We only need to run this for files directly in this folder for now. There are some nested JSON files but they aren't covered by Build-OpenData right now. We can update this later if we add coverage.
```suggestion
      - 'src/open-data/*.csv'
      - 'src/open-data/*.json'
```

---

Gotcha